lib/StripeResource.js

```diff
@@ -102,7 +102,71 @@ StripeResource.prototype = {
     };
   },
 
-  _responseHandler(req, callback) {
+  _addHeadersDirectlyToResponse(res, headers) {
+    // For convenience, make some headers easily accessible on
+    // lastResponse.
+    res.requestId = headers['request-id'];
+
+    const stripeAccount = headers['stripe-account'];
+    if (stripeAccount) {
+      res.stripeAccount = stripeAccount;
```

Comment:
Any reason why we don't do something like

`res.stripeAccount = headers['stripe-account'] || res.stripeAccount;`

for all of these?

Reply:
Only a poorly considered stare decisis, fixed! Thanks!